wetmap/src/components/newModals/shopModal/view.tsx

```diff
@@ -2,149 +2,105 @@ import React, { useState, useContext, useEffect, useRef } from 'react';
 import Itinerary from '../../itineraries/itinerary';
```

Comment:
useState, useContext & useEffect are all imported but unused, so they can be removed from the file